recipes/edyn/all/conanfile.py

```diff
@@ -24,83 +29,74 @@ class EdynConan(ConanFile):
         "fPIC": True,
         "floating_type": "float",
     }
-    generators = "cmake", "cmake_find_package_multi"
-
-    @property
-    def _source_subfolder(self):
-        return "source_subfolder"
-
-    @property
-    def _build_subfolder(self):
-        return "build_subfolder"
-
-    def export_sources(self):
-        self.copy("CMakeLists.txt")
-        for patch in self.conan_data.get("patches", {}).get(self.version, []):
-            self.copy(patch["patch_file"])
-
-    def config_options(self):
-        if self.settings.os == "Windows":
-            del self.options.fPIC
-
-    def configure(self):
-        if self.options.shared:
-            del self.options.fPIC
 
     def requirements(self):
-        self.requires("entt/3.9.0")
+        if Version(self.version) < "1.2.0":
+            self.requires("entt/3.9.0")
+        else:
+            self.requires("entt/3.10.1")
 
     @property
     def _compiler_required(self):
         return {
             "gcc": "9.3", # GCC 9.3 started supporting attributes in constructor arguments
+            "clang": "8",
+            "apple-clang": "10",
         }
 
+    def configure(self):
```

Comment:
Please, move configure to the previous position. It follows conan create flow: https://docs.conan.io/en/latest/reference/commands/creator/create.html#methods-execution-order. Also, it's according to the cmake template flow: https://github.com/conan-io/conan-center-index/blob/master/docs/package_templates/cmake_package/all/conanfile.py

Reply:
Moved them around so they all match the template and will do another PR, thanks!